install/install.sql

```diff
@@ -726,3 +735,26 @@ CREATE TABLE IF NOT EXISTS `v3_ipv4_mapping` (
     COLLATE = utf8mb4_unicode_ci;
 
 -- --------------------------------------------------------------------------------
+--
+-- Table structure for player ranking scores `v3_rankings`
+--
+CREATE TABLE IF NOT EXISTS `v3_rankings` (
+    `id`             INT UNSIGNED NOT NULL,
+    `username`       VARCHAR(30) CHARACTER SET ascii NOT NULL,
+    `scores`         DOUBLE NOT NULL,
+    `max_scores`     DOUBLE NOT NULL,
+    `num_races_done` INT UNSIGNED NOT NULL,
+    PRIMARY KEY (`id`),
+    UNIQUE KEY `username` (`username`),
+    KEY `scores` (`scores`),
```

Comment:
Why is the `scores` a key?

Reply:
https://dba.stackexchange.com/questions/13703/get-the-rank-of-a-user-in-a-score-table
You'll have one scan to get the score list, and another scan or seek to do something useful with it. An index on the score column would help performance on large tables.